### DISCUSSION TITLE
Text overlay to covers in grid view

### DISCUSSION THREAD

**Discussion by BeOSR** [mateusz-bak/openreads#738] [Ideas]
**Is your feature request related to a problem? Please describe.**
In grid view, when a cover is available, only the cover is shown.

**Describe the solution you'd like**
Since covers can sometimes be unclear or not show author and/or title, it would be nice to show author and title as a text overlay to the cover. Alternatively, the cover could be made smaller as not to take up all the surface of the card.

**Comment by mateusz-bak** [comment]
Thanks for the input I think this discussion has good ideas how this could be implemented: https://github.com/mateusz-bak/openreads/discussions/766
